### PR TITLE
Feat/create lmel theme

### LINKED ISSUE #1159: Create LMEL Styled-components theme, with inheritence from default DisMoi theme

### PR DESCRIPTION
**Pull request by JalilArfaoui**
Use the FACET env var (`dismoi` or `lmel`) to apply the correct theme (see /libs/facets/theme) in extension and profiles, and add theme knob in Storybook

`lmelTheme` inherits from `dismoiTheme` … you only have to define things that are different @newick 

Closes #1159

### CODE REVIEW

File: src/app/profiles/App/index.tsx [hunk 1]
```diff
@@ -2,25 +2,27 @@ import React from 'react';
 import { Provider } from 'react-redux';
 import { ConnectedRouter } from 'connected-react-router';
 import { ThemeProvider } from 'styled-components';
-
-import theme from 'app/theme';
+import { useTheme } from 'libs/facets/useTheme.hook';
```
Comment (lutangar) [inline src/app/profiles/App/index.tsx:5]: :+1:

File: src/app/profiles/App/index.tsx [hunk 2]
```diff
@@ -2,25 +2,27 @@ import React from 'react';
...
-          <PageContainer>
-            <Pages />
-          </PageContainer>
-        </Background>
-      </ConnectedRouter>
-    </ThemeProvider>
-  </Provider>
-);
+const App = () => {
+  const theme = useTheme();
```
Comment (lutangar) [inline src/app/profiles/App/index.tsx:12]: :+1:

File: src/components/atoms/Title/Title.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import styled from 'styled-components';
-import { Theme } from '../../../app/theme';
+import { Theme } from '../../../libs/facets/theme';
```
Comment (lutangar) [inline src/components/atoms/Title/Title.ts:2]: I'm not convinced `Theme` definition should live nor in facets nor in libs.... could it be kept in components?
Could our components decently avoid depending on our libs? Might not be practical tho'

**Review by lutangar — APPROVED** [review]
Nice one :+1:

File: src/components/atoms/Title/Title.ts (hunk 3 above)
Comment (lutangar) [reply src/components/atoms/Title/Title.ts:2]: Also, prefer an absolute import over a relative one.

Comment (JalilArfaoui) [reply src/components/atoms/Title/Title.ts:2]: If we go full lib style, we could have 
- /libs/facet <- /components depends on this lib
- /libs/dismoiFacet <- depends on /libs/facet
- /libs/lmelFacet <- depends on /libs/facet

but for a few files and 2 themes, I thougth I’ll keep it simple for now …

Comment (JalilArfaoui) [reply src/components/atoms/Title/Title.ts:2]: relative import fixed

**Pull request merged**